ampersand-model.js

```diff
@@ -20,6 +24,31 @@ var wrapError = function (model, options) {
     };
 };
 
+// recursive function to find the attrs in the serialized object
+var transformForPatch = function transformForPatch(obj, attrs) {
+
+    var attrKeys = keys(attrs);
+
+    if (!isObject(obj)) {
```

Comment:
Super nitpicky performance tweak would be to move this before the `keys(attrs)` line above.